ui/app/helpers/all-icons.js

```diff
@@ -0,0 +1,18 @@
+import { helper } from '@ember/component/helper';
+
+// Generated at compile-time by ember-inline-svg
+import SVGs from '../svgs';
+
+/**
+ * Icons array
+ *
+ * Usage: {{#each (all-icons) as |icon|}}
+ *
+ * Returns the array of all icon strings available to {{x-icon}}. This is a bit of a hack
+ * since the above SVGs import isn't available in the Storybook context.
```

Comment:
I wonder if there is some Storybook configuration that _would_ make it importable, but this is fine with me.

Reply:
Yeah, I almost fell down the "how does storybook even" hole but I stopped myself. There are instances where we're importing files from `/app` but since this file is generated as part of the ember build, it's not available to Storybook when it builds. 

It would have to somehow run the ember addon build scripts which sounds like a broccoli + webpack quagmire.